modules/nominees-election/src/lib.rs

```diff
@@ -79,7 +91,9 @@ impl BondingLedger {
 					false
 				}
 			})
-			.collect();
+			.collect::<Vec<UnlockChunk>>()
+			.try_into()
+			.expect("Maximum UnlockChunks exceeded");
```

Comment:
The message in expect should be a proof of why this can never fail. In this case it is because filter can only reduce number of elements.

Reply:
or you should just use `retain`